packages/web/src/app/NotFound.tsx

```diff
@@ -1,14 +1,20 @@
-import { useRouteError } from "react-router-dom";
+import { useRouteError } from 'react-router-dom';
+import { NotFoundError } from '../shared/accessControl';
 
 export default function NotFound() {
-  const error: any = useRouteError()
+  const error: any = useRouteError();
   console.error(error);
 
-  return <div className="absolute w-full h-full flex flex-col items-center justify-center">
-    <h1 className="font-bold text-xl mb-6">Woops!</h1>
-    <p className="mb-2">An unknown error has occurred.</p>
-    <p className="italic">
-      {error.statusText || error.message}
-    </p>
-  </div>
-}
\ No newline at end of file
+  const message =
+    error instanceof NotFoundError
+      ? 'Not Found'
+      : error.statusText || error.message;
+
+  return (
+    <div className="absolute w-full h-full flex flex-col items-center justify-center">
+      <h1 className="font-bold text-xl mb-6">Woops!</h1>
+      <p className="mb-2">An unknown error has occurred.</p>
+      <p className="italic">{message}</p>
```

Comment:
It would be good to include a link to the home page.

Reply:
As for the not found page stuff, I'm inclined to clean all of that up in a separate PR. Are you ok if I open a separate issue for the error page?

Reply:
Sounds good 👍🏽

Reply:
I don't see the new issue?